### ISSUE TITLE
Prefer Swift constructors raises error from compiler

### ISSUE THREAD

**Issue by squarefrog** [realm/SwiftLint#275]
If you use Prefer Swift constructors (#242), then this will raise a compiler error:

```swift
// This triggers the Prefer Swift constructors rule
let range = NSMakeRange(0, 12)
```

```swift
// Now this passes lint, but the compiler raises an error:
// Extraneous argument labels 'loc:len:' in call
let range = NSMakeRange(loc: 0, len: 12)
```

**Comment by ikesyo** [comment]
`NSRange(location: 0, length: 12)`

**Comment by squarefrog** [comment]
Well spotted, thanks.
